sdk/core/allocator/alloc.h

```diff
@@ -891,6 +902,46 @@ class MState
 	size_t heapFreeSize;
 	size_t heapQuarantineSize;
 
+	/**
+	 * The number of entries currently in the `hazardQuarantine` array.
+	 */
+	size_t hazardQuarantineSize = 0;
```

Comment:
```suggestion
	size_t hazardQuarantineOccupancy = 0;
```

since you use `hazardQuarantineSize` for the actual size of the reserved region, below?